lib/Zonemaster/LDNS/Packet.pm

```diff
@@ -26,6 +26,48 @@ sub data {
     return $self->wireformat;
 }
 
+sub answer {
```

Comment:
Is this method replacing a method of the same name? To me it looks like a new method. Is it used anywhere? Will i magically be used in Zonemaster::Engine?

Reply:
Yes, this changes the meaning of Zonemaster::LDNS::Packet::answer to filter out incomplete RRs. Zonemaster::Engine will get this new behavior instead when it continues to use the Zonemaster::LDNS::Packet::answer method.